### PR TITLE
Feature/ow 352

### PR DESCRIPTION
**Pull request by remi-deltombe-vonage**
OpenTok implementation of audioImpactingVideo

### CODE REVIEW

File: examples/providers/opentok/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    VolumeExtractorTransformer,
+} from "./transformers/volume_extractor_transformer";
+import { VolumeHistogramTransformer } from "./transformers/volume_histogram_transformer";
+import { vec3 } from "./types";
+import { bindButtonToLink, bindSwitch, bindSlider } from "./utils/ui";
+
+const DEFAULT_MIN_VOLUME: number = 0.0001;
+const DEFAULT_MAX_VOLUME: number = 0.01;
+const DEFAULT_COLOR: vec3 = [255, 0, 0];
+const API_KEY = "47568411";
```
Comment (gmininberg) [inline examples/providers/opentok/src/main.ts:18]: lets not put real keys here.. just fake one.

Comment (remi-deltombe-vonage) [reply examples/providers/opentok/src/main.ts:18]: But the app won't work then

File: examples/providers/opentok/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "devDependencies": {
+        "typescript": "^4.6.4",
+        "vite": "^3.1.0",
+        "vite-plugin-glsl": "^0.3.0"
+    },
+    "dependencies": {
+        "@opentok/client": "^2.23.4",
+        "@vonage/media-processor": "^2.0.0",
+        "@vonage/ml-transformers": "^3.1.1",
```
[the remark this answers is not in the document]
Comment (remi-deltombe-vonage) [reply examples/providers/opentok/package.json]: ok

File: examples/providers/opentok/src/main.ts (hunk 1 above)
Comment (gmininberg) [reply examples/providers/opentok/src/main.ts:18]: in the readme file we tell them to replace it...
this key can be used for free video calls LOL

Comment (gmininberg) [reply examples/providers/opentok/src/main.ts:18]: and anyway this token is expired so we are safe... but lets not push tokens (mostly not to public repos)

**Comment by gmininberg** [comment]
i used new token but it seem to do nothing Screen Shot 2022-10-25 at 15 40 41

also we need to provide different headline to the page

**Review by gmininberg — APPROVED** [review]

**Pull request merged**